pkg/webhook/server.go

```diff
@@ -79,6 +101,18 @@ type Server struct {
 
 	// WebhookMux is the multiplexer that handles different webhooks.
 	WebhookMux *http.ServeMux
+}
+
+// New constructs a new Server from the provided options.
+func New(o Options) Server {
```

Comment:
Should this be called NewServer to end up with webhook.NewServer?

Reply:
Good idea, renamed it